### ISSUE TITLE
Training Code

### ISSUE THREAD

**Issue by TakeoffC** [SpiritAshes/Channel-Enhancement#2]
I wanted to inquire if it would be possible for you to provide the training code for the method mentioned in your paper.

**Comment by SpiritAshes** [comment]
Due to my current research focus on the field of object detection, I haven't released the complete training code for this project. However, I will provide a project that includes the training code with CE in the future.

If you urgently need the training code to complete your task, you can try replacing the backbone network and loss function using the works at [](https://github.com/eric-yyjau/pytorch-superpoint) and [](https://github.com/rpautrat/SuperPoint.)

Furthermore, this is our earlier work. Perhaps the recent work of Meta AI, SiLK (https://github.com/facebookresearch/silk), could be more helpful to you, as we have noticed that it covers a comprehensive range of content.

Thank you for your attention, and if you have any further questions, please feel free to continue the discussion.

**Comment by SpiritAshes** [comment]
> Due to my current research focus on the field of object detection, I haven't released the complete training code for this project. However, I will provide a project that includes the training code with CE in the future.
> 
> If you urgently need the training code to complete your task, you can try replacing the backbone network and loss function using the works at and
> 
> Furthermore, this is our earlier work. Perhaps the recent work of Meta AI, SiLK (https://github.com/facebookresearch/silk), could be more helpful to you, as we have noticed that it covers a comprehensive range of content.
> 
> Thank you for your attention, and if you have any further questions, please feel free to continue the discussion.



> Due to my current research focus on the field of object detection, I haven't released the complete training code for this project. However, I will provide a project that includes the training code with CE in the future.
> 
> If you urgently need the training code to complete your task, you can try replacing the backbone network and loss function using the works at and
> 
> Furthermore, this is our earlier work. Perhaps the recent work of Meta AI, SiLK (https://github.com/facebookresearch/silk), could be more helpful to you, as we have noticed that it covers a comprehensive range of content.
> 
> Thank you for your attention, and if you have any further questions, please feel free to continue the discussion.

**Comment by SpiritAshes** [comment]
Due to my current research focus on the field of object detection, I haven't released the complete training code for this project. However, I will provide a project that includes the training code with Cross-Entropy loss in the future.

If you urgently need the training code to complete your task, you can try replacing the backbone network and loss function using the works at (https://github.com/eric-yyjau/pytorch-superpoint) and (https://github.com/rpautrat/SuperPoint).

Furthermore, this is our earlier work. Perhaps the recent work of Meta AI, SiLK (https://github.com/facebookresearch/silk), could be more helpful to you, as we have noticed that it covers a comprehensive range of content.

Thank you for your attention, and if you have any further questions, please feel free to continue the discussion.
